src/ralph/defaults.py

```diff
@@ -99,7 +99,6 @@ def config(key, default_value):
 CONFIG_FILE = APP_DIR / "config.yml"
 CONFIG = load_config(CONFIG_FILE)
 ENVVAR_PREFIX = "RALPH"
-DEFAULT_GELF_PARSER_CHUNCK_SIZE = config("RALPH_DEFAULT_GELF_PARSER_CHUNCK_SIZE", 5000)
 DEFAULT_BACKEND_CHUNCK_SIZE = config("RALPH_DEFAULT_BACKEND_CHUNCK_SIZE", 500)
```

Comment:
Could we (maybe not in this PR) correct the name of this variable?
```suggestion
DEFAULT_BACKEND_CHUNK_SIZE = config("RALPH_DEFAULT_BACKEND_CHUNCK_SIZE", 500)
```

Reply:
Oops, I think I'm the one to blame on this :grimacing:

Reply:
Well spotted) Thanks) I can replace it here)